<ui version="4.0" >
 <class>ProjectDialog</class>
 <widget class="QWidget" name="ProjectDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>704</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="gbGeneral" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" >
      <item row="4" column="1" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Title :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="4" >
       <widget class="QLineEdit" name="leTitle" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Filename :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="4" >
       <widget class="QLabel" name="lFileName" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Author :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" colspan="4" >
       <widget class="QLineEdit" name="leAuthor" />
      </item>
      <item row="6" column="1" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Created :</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Last modified :</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Notes :</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3" colspan="4" >
       <widget class="KDateTimeWidget" name="dtwModified" />
      </item>
      <item row="6" column="3" colspan="4" >
       <widget class="KDateTimeWidget" name="dtwCreated" />
      </item>
      <item row="9" column="3" colspan="4" >
       <widget class="QTextBrowser" name="tbNotes" >
        <property name="readOnly" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="5" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLabel" name="lProjectVersion" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="4" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>LabPlot version :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Project Version :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" >
       <widget class="QLabel" name="lLabPlotVersion" >
        <property name="text" >
         <string>test</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDateTimeWidget</class>
   <extends>QWidget</extends>
   <header>kdatetimewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>selectFile()</slot>
  <slot>updateFileType()</slot>
  <slot>fileInfoDialog()</slot>
 </slots>
</ui>
